stories/topics/con-gcp-application-architecture.adoc

```diff
@@ -1,6 +1,6 @@
 [id="con-gcp-application-architecture"]
 
-= Application architecture
+== Application architecture
```

Comment:
Are you sure about this?

Reply:
After double checking, I reverted this, and update the title in the section below to ensure it will appear at the same level for both AWS/GCP documentation